.gitmodules

```diff
@@ -83,3 +83,6 @@
 	path = externals/mam4xx
 	url = git@github.com:eagles-project/mam4xx.git
 
+[submodule "externals/pybind11"]
```

Comment:
@mahf708 I don't think we should add pip-installable modules as git submodules. Users should be in charge of verifying that dependencies are in place. For standalone testing, we can pip install missing modules locally on the fly.